services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -21,12 +21,15 @@ import { Fragment, useContext } from "react";
 import { Flex, Spinner } from "@chakra-ui/react";
 
 export const AppRoutes = () => {
-  const { userIsAdmin } = useStore().user ?? {};
+  const { userIsAdmin, userIsEndUser } = useStore().user ?? {};
   const { report, contextIsLoaded } = useContext(ReportContext);
 
   // LaunchDarkly
   const mlrReport = useFlags()?.mlrReport;
 
+  // verify whether a user has the required roles / access to MCPAR, MLR, NAAAR reports
+  const userHasAccess = userIsAdmin || userIsEndUser;
```

Comment:
nit: i think it's worth updating this comment mentioning it is possible for a user to be auth'd by idm, but not have either of these, or have the old role that is being removed